### ISSUE TITLE
Deployement works but Inferencing failed for BERT

### ISSUE THREAD

**Issue by vivekjoshi556** [triton-inference-server/tensorrtllm_backend#440]
I have a bert model that I deployed with Triton Inference Server using Tensorrt backend but I am unable to inference using it.

Conversion:
	- Conversion to onxx:
		- optimum-cli export onnx --model distilbert_v1 distilbert_onnx --task text-classification
	- Conversion to Tensorrt Engine:
		- trtexec --onnx=distilbert_onnx/model.onnx --saveEngine=model.plan --minShapes=input_ids:1x1,attention_mask:1x1 --maxShapes=input_ids:8x512,attention_mask:8x512 --optShapes=input_ids:8x512,attention_mask:8x512

Deployment: 
	- For configuration I only mentioned name and backend and I am letting it auto-complete the configuration. 
		On reviewing with tritonclient the config looks something like this: 
			=====================================
			bert Config
			=====================================
			Input Format: [{'name': 'input_ids', 'data_type': 'TYPE_INT32', 'format': 'FORMAT_NONE', 'dims': [-1], 'is_shape_tensor': False, 'allow_ragged_batch': False, 'optional': False}, {'name': 'attention_mask', 'data_type': 'TYPE_INT32', 'format': 'FORMAT_NONE', 'dims': [-1], 'is_shape_tensor': False, 'allow_ragged_batch': False, 'optional': False}]
			
			Output Format: [{'name': 'logits', 'data_type': 'TYPE_FP32', 'dims': [5], 'label_filename': '', 'is_shape_tensor': False}]
		
Sending Request:
	I am using requests python package to send the request like this:
`
import requests
import json

url = "http://0.0.0.0:8000/v2/models/bert/generate"
payload = {
    "input_ids": [101, 2054, 2003, 13012, 2669, 8241, 1029, 102],
    "attention_mask": [1, 1, 1, 1, 1, 1, 1, 1],
}

headers = {
    'Content-Type': 'application/json'
}

%time response = requests.request("POST", url, headers=headers, data=json.dumps(payload))
`

Error: 
{'error': "Unable to parse 'data': Shape does not match true shape of 'data' field"}

**Comment by byshiue** [comment]
Could you share what guide do you follow?
